src/main/kotlin/com/transportes/services/AuthService.kt

```diff
@@ -23,12 +23,14 @@ class AuthService {
     @Autowired private lateinit var passwordEncoder: PasswordEncoder
 
     fun authenticate(email: String, password: String): LoginResponseDTO {
+        val normalizedEmail = email.trim().lowercase()
```

Comment:
```suggestion
        email = email.trim().lowercase()
```
Si en vez de crear una nueva variable, pisas el parametro, no es necesario que modifiques el resto del metodo.